heron/ui/src/python/main.py

```diff
@@ -134,14 +136,15 @@ def main():
 
   # log additional information
   command_line_args = vars(parsed_args)
-  address = socket.gethostbyname(socket.gethostname())
+
+  address = '0.0.0.0' if command_line_args['address'] is None else command_line_args['address']
```

Comment:
ok~ it did listen to 0.0.0.0 and ::
I think the problem right now is if want to force address to socket.gethostbyname(socket.gethostname()) by default which always send Address into Tornado bind.